include/bluetooth/bluetooth.h

```diff
@@ -775,6 +827,60 @@ int bt_le_ext_adv_get_info(const struct bt_le_ext_adv *adv,
 typedef void bt_le_scan_cb_t(const bt_addr_le_t *addr, int8_t rssi,
 			     uint8_t adv_type, struct net_buf_simple *buf);
 
+/** @brief Set or update the periodic advertising parameters.
+ *
+ *  The periodic advertising parameters can only be set or updated on an
+ *  extended advertising set which is neither scannable, connectable or
+ *  anonymous.
+ *
+ *  @param adv   Advertising set object.
+ *  @param param Advertising parameters.
+ *
+ *  @return Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_set_param(struct bt_le_ext_adv *adv,
+			    const struct bt_le_per_adv_param *param);
+
+/** @brief Set or update the periodic advertisement data.
+ *
+ *  The periodic advertisement data can only be set or updated on an
+ *  extended advertising set which is neither scannable, connectable or
+ *  anonymous.
+ *
+ *  @param adv       Advertising set object.
+ *  @param ad        Advertising data.
+ *  @param ad_len    Advertising data length.
+ *
+ *  @return          Zero on success or (negative) error code otherwise.
+ */
+
+int bt_le_per_adv_set_data(const struct bt_le_ext_adv *adv,
+			   const struct bt_data *ad, size_t ad_len);
+
+/** @brief Starts periodic advertising.
+ *
+ *  Enabling the periodic advertising can be done independently of extended
+ *  advertising, but both periodic advertising and extended advertising
+ *  shall be enabled before any periodic advertisement data is sent.
+ *
+ *  @param adv      Advertising set object.
+ *
+ *  @return         Zero on success or (negative) error code otherwise.
+ */
+int bt_le_per_adv_start(struct bt_le_ext_adv *adv);
+
+/** @brief Stops periodic advertising.
+ *
+ *  Disabling the periodic advertising can be done independently of extended
```

Comment:
But what happens if you disable extended advertising? That does disable periodic advertising doesn't it? Please clarify

Reply:
No, periodic advertising will remain active until explicitly disabled. A disabled extended advertising cannot be removed if an active periodic advertising is present.

Reply:
I'm been unsure about this myself. From the spec:
> Disabling the advertising set identified by the Advertising_Handle[i] parameter does not disable any periodic advertising associated with that set.

> If the advertising set is not currently enabled (see the HCI_LE_Set_Extended_Advertising_Enable command), the periodic advertising is not started until the advertising set is enabled. Once the advertising set has been enabled, the Controller shall continue periodic advertising until the Host issues an HCI_LE_Set_Periodic_Advertising_Enable
command with Enable set to 0x00 (periodic advertising is disabled). Disabling the advertising set has no effect on the periodic advertising once the advertising set has been enabled.

So once the periodic advertising (PA) has been enabled, by enabling both, the PA will continue until explicitly disabled, regardless of Ext Adv state.

I'll add some more clarification to the description.